k2/python/k2/fsa_algo.py

```diff
@@ -87,9 +87,12 @@ def intersect(a_fsa: Fsa, b_fsa: Fsa) -> Fsa:
       if and only if the two input FSAs are single FSAs;
       otherwise, len(out_fsa.shape) is 3.
     '''
+    properties_a = a_fsa.properties if a_fsa.properties is not None else -1
```

Comment:
Actually I think a_fsa.properties can never be None.  There is a getter that will compute it.

Reply:
ok